README.md

```diff
@@ -2,7 +2,7 @@
 
 Fork of Endilll's [vapoursynth-preview](https://github.com/Endilll/vapoursynth-preview) (not maintained anymore)
 
-This program is meant to be paired with a code editor with integrated terminal like Visual Studio Code.
+This program is meant to be paired with a code editor with integrated terminal like VS Code or Vim.
```

Comment:
why did you feel the need to shorten visual studio as vs? that can be confused with vapoursynth